package.json

```diff
@@ -166,7 +166,6 @@
     "contracts:compile:v3": "typechain --target ethers-v5 --out-dir src/types/v3 \"./node_modules/@uniswap/**/artifacts/contracts/**/*.json\"",
     "contracts:compile": "yarn contracts:compile:abi && yarn contracts:compile:v3",
     "graphql:generate": "graphql-codegen --config codegen.yml",
-    "prei18n:extract": "touch src/locales/en-US.po",
```

Comment:
Fixes an issue for developers working on Windows. The `touch` command does not exist in Windows, so running `yarn` will fail. 

From what I can see, it is not necessary to actually run this command, as `src/locales/en-US.po` is already checked into the repo and therefore doesn't need to be created via `touch`. If there is a specific reason why this command does need to be run as part of the install process, I would recommend something like this:

```json
"prei18n:extract": "node prei18n-extract.js",
```

```js
// prei18n-extract.js
import { exec } from 'child_process'
const isWindows = process.platform === 'win32' || /^(msys|cygwin)$/.test(process.env.OSTYPE)

if (isWindows) {
  exec(`type nul > src/locales/en-US.po`)
} else {
  exec(`touch src/locales/en-US.po`)
}
```

Closes: https://github.com/Uniswap/interface/issues/2442

Reply:
@zzmp thoughts?

Reply:
Added in https://github.com/Uniswap/interface/pull/3728